src/app/home/work-item-widget/work-item-widget.component.ts

```diff
@@ -62,45 +64,44 @@ export class WorkItemWidgetComponent implements OnDestroy, OnInit  {
    */
   private fetchWorkItemsBySpace(space: Space): void {
     this.currentSpace = space;
-    this.subscriptions.push(this.userService
-      .getUser()
-      .do(() => this.loading = true)
-      .do(() => this.workItemService._currentSpace = space)
-      .do(() => this.workItemService.buildUserIdMap())
-      .switchMap(() => this.userService.loggedInUser)
```

Comment:
Actually the more I think about it, would it be possible to bring back the `userService.loggedInUser` into the observable chain, similar to how it is here? There is no guarantee that the constructors subscription to retrieve the `loggedInUser` has completed by the time the queries would be created (and thus `loggedInUser` may not contain a value), and it may be safer to handle it in sequence along with fetching the work items.

Reply:
@aptmac But that was the issue here. After the logout `loggedInUser` was undefined since it was an observable which caused the problem.

Reply:
Hm right, ignore that then haha.